FakeItEasy.sln

```diff
@@ -1,26 +1,32 @@
-﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0

```

Comment:
Sure, but this is also the current situation. The current sln on master can be opened by anyone using VS2015. Indeed, on my main dev machine I *only* have VS2015.

If we want to restrict people to C#5 then that is done in the project file like so - https://github.com/Particular/NServiceBus/commit/ab95b4f5a00fe9c2866721756671b1aa6d82e000#diff-f94eb20da3f228e8c172d40dd6cab771

We can address that in another PR.